public_src/services/overpass.service.ts

```diff
@@ -754,4 +764,41 @@ export class OverpassService {
       throw new Error(JSON.stringify(err));
     });
   }
+
+  private downloadMultipleNodeData(toDownload: any): any {
+    let requestBody = `
+      [out:json][timeout:25];
+      (
+         node(id:${toDownload.join(', ')});
+      );
+      (._;<;);
+      out meta;`;
+    console.log('query', requestBody);
+    requestBody = this.replaceBboxString(requestBody.trim());
+    this.httpClient
+      .post(ConfService.overpassUrl, requestBody, { headers: Utils.HTTP_HEADERS })
+      .subscribe(
+        (res) => {
+          if (!res) {
+            return alert('No response from API. Try to select element again please.');
+          }
+
+          console.log('res', res);
+          for (const element of toDownload) {
+            this.storageSrv.elementsDownloaded.add(element);
+          }
+
+          for (const element of res['elements']) {
+            if (!this.storageSrv.elementsMap.has(element.id)) {
+              this.storageSrv.elementsMap.set(element.id, element); }
+          }
+
+          this.appActions.actSetErrorCorrectionMode('menu');
+          this.errorHighlightSrv.isDataDownloaded.emit(true);
+        },
+        (err) => {
+          this.warnSrv.showError();
+          throw new Error(err.toString());
```

Comment:
` err.toString()` actually just logged `[object] [object]`. Can you set it to `JSON.stringify(err)`?